content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -56,14 +56,16 @@ Variable | Description
 -------- | -----------
 `CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
 `CTX_REMOTE_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the remote cluster.
-`EXTERNAL_ISTIOD_ADDR` | The hostname for the ingress gateway on the external control plane cluster. This is used by the `remote_cluster` to access the external control plane.
+`REMOTE_CLUSTER_NAME` | The name of the remote cluster.
+`EXTERNAL_ISTIOD_ADDR` | The hostname for the ingress gateway on the external control plane cluster. This is used by the remote cluster to access the external control plane.
 `SSL_SECRET_NAME` | The name of the secret that holds the TLS certs for the ingress gateway on the external control plane cluster.
 
-Set the `CTX_EXTERNAL_CLUSTER` and `CTX_REMOTE_CLUSTER` now. You will set the others later.
+Set the `CTX_EXTERNAL_CLUSTER`, `CTX_REMOTE_CLUSTER`, and `REMOTE_CLUSTER_NAME` now. You will set the others later.
 
 {{< text bash >}}
 $ export CTX_EXTERNAL_CLUSTER=external_cluster
 $ export CTX_REMOTE_CLUSTER=remote_cluster
+$ export REMOTE_CLUSTER_NAME=remote_cluster
```

Comment:
minor nit: Since you have `remote_cluster` for both items, should you make them `external_/remote_cluster_ctx` and remote_cluster_name?

Reply:
The `CTX_` names are a consistent naming convention used in the multicluster docs, so we should leave them that way. Doing the opposite, `NAME_REMOTE_CLUSTER`, sounds awkward. So, I'd leave them as they are.

Reply:
My comment was more of the values on the right side of the `=` sign. You have remote_cluster twice for different variables. You probably want to highlight to the reader that these values can be different.

Reply:
Oh, I see. I think it's fine. Usually the cluster name and context name are the same, but not always. That's why we need two variables. Users should be able to figure it out, given the table, above, with clear descriptions of all the vars.